controllers/storage/minio.go

```diff
@@ -41,21 +51,30 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 		return m.ProvisionExternalStorage()
 	}
 
-	err := m.KubeClient.Get(m.Ctx, m.getminIONamespacedName(), &minioCR)
+	err := m.KubeClient.Get(m.getminIONamespacedName(), &minioCR)
 	if k8serror.IsNotFound(err) {
-		// TODO need test
 		return m.Provision()
 	} else if err != nil {
 		return minioNotReadyStatus(ErrorReason0, err.Error()), err
 	}
 
+	var minioStatefulSet appsv1.StatefulSet
+	err = m.KubeClient.Get(m.getminIONamespacedName(), &minioStatefulSet)
+	if err != nil {
+		return minioNotReadyStatus(ErrorReason0, err.Error()), err
+	}
+
+	if minioStatefulSet.Status.ReadyReplicas == m.HarborCluster.Spec.Storage.InCluster.Spec.Replicas {
+		// TODO create default bucket
+		return m.ProvisionInClusterSecret(&minioCR)
+	}
 	return nil, nil
```

Comment:
what does `nil, nil` mean here? Do nothing? Waiting for the next round reconcile process?

Reply:
here mean minio is creating, continue Waiting for creating successful.